### PR TITLE
Integration's description docs

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
Ready

### CODE REVIEW

**Comment by glicht** [comment]
Make sure to add the doc to sidebars.js. Add it after the doc format document. See https://github.com/demisto/content-docs/blob/master/sidebars.js#L89

**Comment by glicht** [comment]
@guyfreund I updated the `:::note` for better formatting. 

Please add an examples section as we do in the integration readme docs.

**Review by ShahafBenYakir — APPROVED** [review]

**Review by ShahafBenYakir — CHANGES_REQUESTED** [review]
Wait

**Comment by ShahafBenYakir** [comment]
@glicht I added the document to the sidebar and added examples. see this:

Integration Description File | Cortex XSOAR 2020-09-16 18-53-25

File: docs/integrations/description-docs.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+id: description-docs
+title: Integration Description File
+---
+
+The integration description file is designated to help the XSOAR users to easily configure an instance of the chosen integration.
+
+## Path
+The description file should be placed along with all of the integraion's files.
```
Comment (fvigo) [inline docs/integrations/description-docs.md:9]:
```suggestion
The description file should be placed along with all of the integration's files. 
```

File: docs/integrations/description-docs.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+~/.../content/Packs/HelloWorld/Integrations/HelloWorld/HelloWorld_description.md
+```
+
+
+## Data
+The description file should contain details on how to configure an instance of the integration, together with the relevant details needed from the product that are important to perform an easy configuration.
+The file's content can include troubleshooting tips and advanced details for different configuration cases.
+
+:::note 
+This should not be confused with integration README.md which is the integration documentation document.
```
Comment (fvigo) [inline docs/integrations/description-docs.md:22]: Please add a link to the integration-docs document

**Review by fvigo — CHANGES_REQUESTED** [review]
Can you please add a link to this document in the Integration documentation doc?

File: docs/integrations/description-docs.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+~/.../content/Packs/HelloWorld/Integrations/HelloWorld/HelloWorld_description.md
+```
+
+
+## Data
+The description file should contain details on how to configure an instance of the integration, together with the relevant details needed from the product that are important to perform an easy configuration.
+The file's content can include troubleshooting tips and advanced details for different configuration cases.
+
+:::note 
+This should not be confused with integration README.md which is the [Integration Documentation document](https://xsoar.pan.dev/docs/integrations/integration-docs).
```
Comment (ShahafBenYakir) [inline docs/integrations/description-docs.md:22]:
```suggestion
This should not be confused with integration README.md which is the [Integration Documentation] document (https://xsoar.pan.dev/docs/integrations/integration-docs).
```

File: docs/integrations/integration-description.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+~/.../content/Packs/HelloWorld/Integrations/HelloWorld/HelloWorld_description.md
+```
+
+
+## Data
+The description file should contain details on how to configure an instance of the integration, together with the relevant details needed from the product that are important to perform an easy configuration.
+The file's content can include troubleshooting tips and advanced details for different configuration cases.
+
+:::note 
+This should not be confused with integration README.md which is the [Integration Documentation](https://xsoar.pan.dev/docs/integrations/integration-docs) document.
```
Comment (fvigo) [inline docs/integrations/integration-description.md:22]:
```suggestion
This should not be confused with the integration README file, documented [here](../integrations/integration-docs).
```

**Review by fvigo — APPROVED** [review]
Great, just two minor suggestions

File: docs/integrations/integration-docs.md [hunk 5]
```diff
@@ -15,6 +15,10 @@ Documentation is a critical step that assists customers who may use your integra
 The documentation must conform to the [Integration Doc Format](doc-structure).
```
Comment (ShahafBenYakir) [inline docs/integrations/integration-docs.md:15]: @fvigo [This](https://deploy-preview-291--demisto-content-docs.netlify.app/docs/integrations/integration-docs) leads to nowhere... any can we remove this?

Comment (fvigo) [reply docs/integrations/integration-docs.md:15]: Yes, good catch. We can maybe add an example of HelloWorld instead: https://github.com/demisto/content/blob/master/Packs/HelloWorld/Integrations/HelloWorld/README.md

Comment (ShahafBenYakir) [reply docs/integrations/integration-docs.md:15]: @fvigo I think that there are enough example here https://deploy-preview-291--demisto-content-docs.netlify.app/docs/integrations/integration-docs#documentation-examples

Comment (fvigo) [reply docs/integrations/integration-docs.md:15]: ok

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**